packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphComponent/index.tsx

```diff
@@ -238,6 +241,22 @@ export default function FlameGraphComponent(props: FlamegraphProps) {
         );
       };
 
+      const FitModeItem = () => {
+        const isHeadFirst = fitMode === HeadMode;
+
+        const handleClick = () => {
+          const newValues = isHeadFirst ? TailMode : HeadMode;
+          updateFitMode(newValues);
+        };
+
+        return (
+          <MenuItem key="fit-mode" onClick={handleClick}>
+            {/* todo: add icons */}
+            Fit mode: {isHeadFirst ? 'tail first' : 'heat first'}
```

Comment:
```suggestion
            Fit mode: {isHeadFirst ? 'tail first' : 'head first'}
```

Although since we have enough space, I think we can be more explicit `Fit {head, tail} first` or or some better wording cc @Rperry2174

Reply:
I think the wording is fine, but for consistency we should add the icon the same way the other options have an icon

Reply:
absolutely..which one should i use ?

Reply:
Lets use the same icons from the toolbar (depending on whether it says "head" or "tail" first. As far as the text for the other ones we kind of explain what happens with the option:
<img width="348" alt="image" src="https://user-images.githubusercontent.com/23323466/201460227-c2732694-b37e-4563-9826-0b031c0b2aa7.png">


What do you guys think about:

- "[ Icon ] Show text head first" 
- "[ Icon] Show text tail first"

Reply:
looks good i think

https://user-images.githubusercontent.com/47758224/201632259-37090c6c-1bed-447f-86f8-da2597501bf8.mov